apollo-biz/src/test/java/com/ctrip/framework/apollo/biz/service/NamespaceBranchServiceTest.java

```diff
@@ -82,7 +82,15 @@ public void testUpdateBranchGrayRulesWithUpdateOnce() {
     Assert.assertEquals(ReleaseOperation.APPLY_GRAY_RULES, releaseHistory.getOperation());
     Assert.assertEquals(0, releaseHistory.getReleaseId());
     Assert.assertEquals(0, releaseHistory.getPreviousReleaseId());
-    Assert.assertTrue(releaseHistory.getOperationContext().contains(rule.getRules()));
+    Assert.assertTrue(containRule(releaseHistory.getOperationContext(),rule.getRules()));
+  }
+
+  public boolean containRule(String context, String rule){
+    boolean contains = true;
+    String[] rules = rule.substring(2,rule.length()-2).split(",");
```

Comment:
I think use [GrayReleaseRuleItemTransformer#batchTransformFromJSON](https://github.com/apolloconfig/apollo/blob/301d66dcc28b2d698eefed3ecc9aa3273afb9ddf/apollo-common/src/main/java/com/ctrip/framework/apollo/common/utils/GrayReleaseRuleItemTransformer.java#L35) parse  rules might be better. What do you think?